pyrelational/pipeline/pipeline.py

```diff
@@ -110,8 +113,9 @@ def current_performance(self, test_loader: Optional[DataLoader] = None, query: O
         result = self.model.test(self.test_loader if test_loader is None else test_loader)
         result = self.compute_hit_ratio(result, query)
         self.performances[self.iteration] = result
+        return None
 
-    def compute_hit_ratio(self, result: Dict, query: Optional[List[int]] = None) -> Dict:
+    def compute_hit_ratio(self, result: Dict[str, float], query: Optional[List[int]] = None) -> Dict[str, float]:
```

Comment:
Maybe use RETURN_TYPE here?

Reply:
(is RETURN TYPE actually necessary though, Dict[str, float] doesn't seem that long?)

Reply:
good point, it used to be more complex I believe